FiveCalls/FiveCalls/IssueDetailViewController.swift

```diff
@@ -44,8 +44,12 @@ class IssueDetailViewController : UIViewController, IssueShareable {
             tableView.deselectRow(at: indexPath, animated: true)
         }
         logs = ContactLogs.load()
-        
-        navigationController?.setNavigationBarHidden(false, animated: true)
+
+        if let nav = self.navigationController, nav.viewControllers.count > 1 {
+            navigationController?.setNavigationBarHidden(false, animated: true)
+        } else {
+            tableView.contentInset = UIEdgeInsets(top: 64, left: 0, bottom: 0, right: 0)
```

Comment:
'cause that's not a navbar above it; it's the shared header from the Issues List page. I tried to change as little as possible, so I'm still using a lot of the same architecture from the iPhone version.